### ISSUE TITLE
Unsigned Subclipse Packages [Tigris #1525]

### ISSUE THREAD

**Issue by markphip** [subclipse/archive#89]
## Description

```
When I try to install Subclipse [version 1.10.0] Eclipse warns me that the 
packages are unsigned:

Security Warning

Warning: You are installing software that contains unsigned content. The 
authenticity or validity of this software cannot be established. Do You want to 
continue with this installation?

If I click on the [Details >>] button the following packages appear:

plugins\com.collabnet.subversion.merge_4.0.0.jar
features\com.collabnet.subversion.merge.feature_4.0.0
plugins\org.tigris.subversion.clientadapter_1.10.0.jar
features\org.tigris.subversion.clientadapter.feature_1.10.0
plugins\org.tigris.subversion.clientadapter.javahl_1.8.0.jar
features\org.tigris.subversion.clientadapter.javahl.feature_1.8.0
plugins\org.tigris.subversion.clientadapter.javahl.win64_1.8.0
plugins\org.tigris.subversion.subclipse.core_1.10.0.jar
features\org.tigris.subversion.subclipse_1.10.0
plugins\org.tigris.subversion.subclipse.ui_1.10.0.jar

Basically all of the packages related to Subclipse are unsigned.

As far as I know this behavior happened in the previous versions of Subclipse 
[1.6 and 1.8 as far as I remember].

Could you guys please sign the packages when you publish them :) ?
Thank you in advance !
```
## Metadata Imported from Tigris (Issue 1525)
- **Creation Date**: 2013-06-23 06:33:24
- **Reporter**: tatacalu
- **Subcomponent**: core
- **Version**: 1.10.x
- **Milestone**: not determined
- **Keywords**: 
- **Cc**:
